Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -157,7 +161,7 @@ import BraintreeDataCollector
     
     // MARK: - Internal Methods
     
-    func handleBrowserSwitchReturn(
+    func handleReturn(
```

Comment:
Open to alternate names for this as well since we have a lot of `handleReturn/handleReturnURL` related names already. This method is used to construct our request to `v1/payment_methods/paypal_accounts`.

Reply:
The "handle__" functions sometimes require me to dig around to see what they do. 
I can't think of a great name but could we include a short description on top
like "parses url to list to v1/payment_methods/payPal_accounts endpoint or do u think
that's too much for non public function?

Reply:
And what determines approvalURL.redirectType being
.paPalApp vs .webBrowser?

Reply:
I also can't think of one, but going to think more on it today and if other folks have options this is open for opinions. I was initially thinking something along the lines of `parseReturnURL`, but again not _much_more details compared to `handleReturn`. Will keep thinking on it.

Reply:
> And what determines approvalURL.redirectType being .payPalApp vs .webBrowser?

This is determined based on the logic in PR #1234